graph/schema.resolvers.go

```diff
@@ -10,21 +10,39 @@ import (
 	"fmt"
 	"log"
 
+	"github.com/Matterhorn-Apps/MatterhornApiService/auth"
 	"github.com/Matterhorn-Apps/MatterhornApiService/database"
 	"github.com/Matterhorn-Apps/MatterhornApiService/graph/generated"
 	"github.com/Matterhorn-Apps/MatterhornApiService/graph/model"
 )
 
+func (r *mutationResolver) CreateMe(ctx context.Context) (*model.User, error) {
+	// Get the user ID from context
+	// If not found, request is not authenticated and should fail
+	tokenSub, err := auth.GetUserIDFromContext(ctx)
+	if tokenSub == nil || err != nil {
+		return nil, errors.New("request failed: caller not authenticated")
+	}
+
+	// Try to get user with given token sub as ID
+	// This will fail if the user already exists
```

Comment:
if the user does already exist, what response is the client going to get?

Reply:
Error response. (Line 45)